app-modules/portal/src/Http/Controllers/KnowledgeManagementPortal/KnowledgeManagementPortalController.php

```diff
@@ -57,10 +57,10 @@ public function show(): JsonResponse
 
         return response()->json([
             'layout' => $settings->knowledge_management_portal_layout ?? PortalLayout::Full,
-            'header_logo' => $logo?->getTemporaryUrl(
+            'header_logo' => $logo ? $logo?->getTemporaryUrl(
```

Comment:
Because you are ensuring that `$logo` is not falsy, you now no longer need the `null` safety here.

```suggestion
            'header_logo' => $logo ? $logo->getTemporaryUrl(
```